bin/win/x64/README.md

```diff
@@ -0,0 +1,142 @@
+<div align="center">
+    <a href="https://php.net">
+        <img
+            alt="PHP"
+            src="https://www.php.net/images/logos/new-php-logo.svg"
+            width="150">
+    </a>
+</div>
+
+# The PHP Interpreter
+
+PHP is a popular general-purpose scripting language that is especially suited to
+web development. Fast, flexible and pragmatic, PHP powers everything from your
+blog to the most popular websites in the world. PHP is distributed under the
+[PHP License v3.01](LICENSE).
+
+[![Push](https://github.com/php/php-src/actions/workflows/push.yml/badge.svg)](https://github.com/php/php-src/actions/workflows/push.yml)
+[![Build status](https://travis-ci.com/php/php-src.svg?branch=master)](https://travis-ci.com/github/php/php-src)
+[![Build status](https://ci.appveyor.com/api/projects/status/meyur6fviaxgdwdy/branch/master?svg=true)](https://ci.appveyor.com/project/php/php-src)
+[![Build Status](https://dev.azure.com/phpazuredevops/php/_apis/build/status/php.php-src?branchName=master)](https://dev.azure.com/phpazuredevops/php/_build/latest?definitionId=1&branchName=master)
+[![Fuzzing Status](https://oss-fuzz-build-logs.storage.googleapis.com/badges/php.svg)](https://bugs.chromium.org/p/oss-fuzz/issues/list?sort=-opened&can=1&q=proj:php)
+
+## Documentation
+
+The PHP manual is available at [php.net/docs](https://php.net/docs).
+
+## Installation
+
+### Prebuilt packages and binaries
+
+Prebuilt packages and binaries can be used to get up and running fast with PHP.
+
+For Windows, the PHP binaries can be obtained from
+[windows.php.net](https://windows.php.net). After extracting the archive the
+`*.exe` files are ready to use.
+
+For other systems, see the [installation chapter](https://php.net/install).
+
+### Building PHP source code
+
+*For Windows, see [Build your own PHP on Windows](https://wiki.php.net/internals/windows/stepbystepbuild_sdk_2).*
+
+For a minimal PHP build from Git, you will need autoconf, bison, and re2c. For
+a default build, you will additionally need libxml2 and libsqlite3.
+
+On Ubuntu, you can install these using:
+
+    sudo apt install -y pkg-config build-essential autoconf bison re2c \
+                        libxml2-dev libsqlite3-dev
+
+On Fedora, you can install these using:
+
+    sudo dnf install re2c bison autoconf make libtool ccache libxml2-devel sqlite-devel
```

Comment:
Are you using Ubuntu/Fedora to create the Windows builds?

Reply:
These builds are downloaded directly from the [PHP website](https://windows.php.net/download/)

That README.md is from their download.

The only modification I made to the downloaded files was the `php.ini` to include required extensions out of the box.

Reply:
Gotcha. It's a little confusing because it looks like you've provided instructions for how to build the Windows binaries provided here.

Reply:
Sure, I could see that if you were looking at only this commit but these files will be buried in the PHP Platform/Architecture directory of this repo so I felt it was best to preserve the original files. 

Also since this repo is intended to be included by other packages and not directly, I felt this was not a problem.

Reply:
What’s the purpose of including this readme?

Reply:
As I mentioned above, I included the _entire_ downloaded distribution.
A markdown file is negligible in size compared to any of the binaries.

Reply:
I should note that users nor developers should really ever been in this directory to see these files.

Reply:
If @simonhamp wanted to remove the README I see no problem with that. 
I see in the PHP license that as long as the license.txt is included we should be good to go but I erred on the side of caution and included all files for now.

I figured since this is the Alpha I would make no attempt yet to optimize the distributed size as this would still give us more options later if we wanted to allow which modules get configured in the php.ini.

If various other libs were missing, this would restrict options at that point. 

The README is not really relevant to this but still, It felt like I would be stripping something that really didn't affect the size anyway.

Reply:
I see what you mean now by including all the files that are part of that download. I had originally thought you were trying to create static binaries for Windows, like this repo does for macOS. Is it possible to create static binaries for Windows, too?

Reply:
Correct! Now we're on the same page 👍🏻 

I believe [this](https://github.com/crazywhalecc/static-php-cli) is where the Mac static binaries were obtained and will be the source of the linux ones as well.

I believe it is in the plans of the crazywalecc repo to include Windows static builds as well but the included build from the PHP Windows page works just fine. 

I also felt from a security perspective, it was more _appropriate_ to use the binaries shipped by PHP.net as opposed to 'manipulating' them somehow. Feels this exposes apps built with these included binaries to more supply chain exposure, etc.

Thanks for reviewing!

Reply:
Several others have mentioned the crazywalecc project to me, but since this project doesn’t mention crazywalecc, I assume they’re using their own scripts to build the static binaries.

Reply:
I spoke with @mpociot (maintainer of the packages that use this package) at LaraconUS in Nashville last week in person and he mentioned that 'a Chinese repo' was being used. I can only assume it is that repo where they were sourced.

Perhaps @simonhamp can confirm?

Reply:
Can confirm the static builds here are done by the static-php-cli repo by crazywhale

That project doesn't yet support static builds for windows, which is proving to be quite tricky to pull off, so this approach is fine for now

As for this README, I think it only adds confusion - while I agree with your reasoning @chrisreedio, we're already seeing inquisitive minds digging deep into all of NativePHP, and trying many variations around what we've documented.

Which is great! But could also be the source of some spurious errors and wild bug reports.

I think it's best if we limit the chances of stuff like that. So I'd recommend removing this file.

Also, while it becomes clear how to build your own static binaries, it's important to note the security and support implications of using and distributing something custom with your apps and so **we strongly recommend that everyone use the binaries provided by this repo for maximum compatibility and ease of distribution**

Reply:
@simonhamp I'll get the README removed for this PR.

Are there any other files you feel I should remove?